chatsky/core/ctx_utils.py

```diff
@@ -64,6 +64,11 @@ class FrameworkData(BaseModel, arbitrary_types_allowed=True):
     "Enables complex stats collection across multiple turns."
     slot_manager: SlotManager = Field(default_factory=SlotManager)
     "Stores extracted slots."
+    response_exception: Optional[str] = Field(default=None, exclude=True)
```

Comment:
Do you remember why this holds repr instead of exception?
Seems like it's better to store exception since this field will not be serialized.